src/plugins/data/public/search/search_strategy/default_search_strategy.test.ts

```diff
@@ -62,18 +63,15 @@ describe('defaultSearchStrategy', function() {
           },
         ],
         esShardTimeout: 0,
-        searchService: {
-          aggs: {
-            calculateAutoTimeExpression: jest.fn().mockReturnValue('1d'),
-          },
+        searchService: ({
           search: newSearchMock,
           __LEGACY: {
             esClient: {
               search: searchMock,
               msearch: msearchMock,
             },
           },
-        },
+        } as unknown) as jest.Mocked<ISearchStart>,
```

Comment:
Changed this to casting because there was a circular dependency going back to require the search.aggs mocks, and it felt unnecessary to duplicate the entire search service contract here